README.md

```diff
@@ -24,18 +24,31 @@ You can use your own computer as a server, but for other people to connect to yo
   [5]: http://en.wikipedia.org/wiki/Port_forwarding
 
 
-Installing
+Previous Requisites
 ------------------------------------------------------------------------
+Supported Node version is latest LTS 6.10.3. You can set the specific node version using:
+- [nvm](https://github.com/creationix/nvm) for Linux or Mac OS
+- [nvm-windows](https://github.com/coreybutler/nvm-windows) for Windows users
 
-    ./pokemon-showdown
+Then you can install this project's node version by running:
+```
+nvm install
+```
 
-(Requires Node.js 6+)
+And from then on you can simply run `nvm use` to use this project's node version
 
+Install dependencies:
+- npm install
+
+Installing
+------------------------------------------------------------------------
+
+    ./pokemon-showdown
 
 Detailed installation instructions
 ------------------------------------------------------------------------
 
-Pokémon Showdown requires you to have [Node.js][6] installed, 6.x or later.
+Pokémon Showdown requires you to have [Node.js][6] installed, 6.x or later. See [Previous Requisites section](#previou-requisites)
```

Comment:
For the record, our convention is to use `[text][1]` `[1]: url` convention in README files. Our README files are structured the way they are so that they're readable by a plaintext editor (which is the main advantage of Markdown in the first place).